### PR TITLE
look for .kic in PWD and parents

### LINKED ISSUE #331: CLI definition
- add "tree" for custom CLI configs
- currently looks in pwd
  - running in myapp works
  - running in myapp/src doesn't get the custom config

### PR DESCRIPTION
**Pull request by bartr**
## Purpose of PR

- Look for .kic (or .flt or .kivm) in PWD and parents
- update ci-cd for releases

## Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [x] CI-CD changes
- [ ] GitHub Template changes

## Issues Closed or Referenced

- Closes #331

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -25,14 +25,24 @@ jobs:
...
           goos: ${{ matrix.goos }}
           goarch: ${{ matrix.goarch }}
           project_path: "./src/kic"
-          binary_name: "akdc"
-          ldflags: "-s -w"
+          binary_name: "flt"
+          ldflags: "-s -w -X 'kic/cmd.TargetCli=flt'"
+
+      # add flt CLI to release
+      - uses: wangyoucao577/go-release-action@v1.25
```
Comment (aflinchb) [inline .github/workflows/release.yaml:41]: nit: comment on line 40 should be add kivm instead of flt

**Review by aflinchb — APPROVED** [review]
Approved w/ minor comment

**Review by wbreza — APPROVED** [review]
LGTM

File: .github/workflows/release.yaml (hunk 1 above)
Comment (bartr) [reply .github/workflows/release.yaml:41]: good catch

**Pull request merged**